src/rf/js/src/home/components/modals.js

```diff
@@ -122,30 +124,50 @@ var UploadModal = React.createBackboneClass({
         return fileNameParts[fileNameParts.length - 1];
     },
 
-    s3BucketKeyValid: function(s3BucketKey) {
-        // example valid s3BucketKey: ab-cd.ef/gh1/i-j.k
-        var re = /^([\w-.]+\/)+[\w-]+\.\w+$/;
-        return re.test(s3BucketKey);
+    isTiff: function(fileName) {
```

Comment:
For uploaded files I don't think this is as robust as our existing solution.
https://github.com/azavea/raster-foundry/blob/develop/src/rf/js/src/core/uploads.js#L65
which should be updated to remove the unallowed types.